gax/call_context.h

```diff
@@ -0,0 +1,150 @@
+// Copyright 2019 Google Inc.  All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAPIC_GENERATOR_CPP_GAX_CALL_CONTEXT_H_
+#define GAPIC_GENERATOR_CPP_GAX_CALL_CONTEXT_H_
+
+#include "grpcpp/client_context.h"
+#include "internal/gtest_prod.h"
+
+#include <chrono>
+#include <functional>
+#include <map>
+#include <string>
+#include <utility>
+
+using GrpcContextPolicyFunc = std::function<void(grpc::ClientContext*)>;
```

Comment:
Can we add a comment to document this signature?

Reply:
Done